### DISCUSSION TITLE
팀 블로그 제작하기

### DISCUSSION THREAD

**Discussion by jgjgill** [prgrms-web-devcourse/React-Playground-Study#37] [Project]
본격적인 블로그 제작에 들어가고자 합니다!!
우선 제가 생각한 부분들을 공유하겠습니다..!

## 이것저것 생각한 것들...

만든 팀 블로그를 팀 프로젝트에도 활용하면 좋겠다는 생각을 가지고 있습니다!
(에러 상황 해결, 팀 규칙 정한 과정, 여러 상황들에 대한 기록, 지속적인 블로그 관리 및 개선)

- 1차 목표는 **31일**
- 구현 리스트
  - 메인 페이지 (팀 소개하기 - 이 과정에서 컴포넌트를 만드는 연습을 할 수 있을 것 같다..! 스토리북의 활용..?)
  - 블로그 페이지 (**포스팅!** <최우선 작업> 게시물 리스트, 특정 게시물 - nextjs에서 ssg를 사용하는 법을 배우자..!)
- 관련 자료
  - [NextJS Blog Template](https://vercel.com/templates?utm_source=next-site&utm_medium=navbar&utm_campaign=learn_basics_data-fetching_setup&framework=next.js&type=blog)
  - [Pre-rendering and Data Fetching](https://nextjs.org/learn/basics/data-fetching)
  - [Search Engine Optimization](https://nextjs.org/learn/seo/introduction-to-seo)
  - [blog-starter](https://github.com/vercel/next.js/tree/canary/examples/blog-starter)

### 디자인

피그마도 미리 적응할 겸 기본 틀을 만들면 좋을까 싶은데 어떤가요?

### 블로그에 대해 이야기하는 시간을 가지자!

언제가 괜찮으신지..?

### 여러분의 생각이 궁금합니다!

현재 블로그에 대해 가지고 있는 생각을 말씀해주세요!
ex) 이런 기능을 하고 싶다!, 디자인은 이렇게 하면 좋을 것 같다!, ~~ 템플릿을 참고하면 좋을 것 같다! 등등등...

**Comment by backward99** [comment]
팀 스크럼 시간에 하면 되겠다라고 생각했는데 시간이 좀 부족할까요..?

docs에 있는 내용들이 블로그에 올라가는 건가요?

**Reply by jgjgill** [reply]
docs에 있는 것들을 블로그 프로젝트 폴더 내로 이동하면 docs에 있는 것도 포함시킬 수 있을 것 같네요!!

**Comment by jgjgill** [comment]
### 컴포넌트를 만들어보는 연습..?

> ssg 방식으로는 md파일을 받고 렌더링만 하면 되기 때문에 팀프로젝트에 적합하지는 않다!

같이 할 수 있는 부분이 뭐가 있을까 고민해봤을 때 팀블로그에 쓰일 컴포넌트들을 만들어보면 좋을 것 같아요!
이를 통해 더 나은 컴포넌트를 만들기? 나만 쓰는게 아닌 확장성있는 컴포넌트? 이러한 부분들을 학습할 수 있지 않을까 싶습니다..!
ex) Card, Button, Tabs, Progress, Heading, 등등등?

- 기술 스택으로는 스토리북을 추가해야 할 것 같습니다!
- [스토리북 관련 컴포넌트 학습](https://www.notion.so/prgrms/12-a1b82f4e66f24f049f31fd813afc5401?pvs=4#ef7ee7c62a6f4a17b03fd7f0793c0315)
